ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -254,6 +255,12 @@ module.exports = function CesiumMap(
     return billboardCollection
   }
 
+  function setupLabel() {
```

Comment:
:pencil2: `setupLabels` or `setupLabelCollection`

Reply:
Should the `setupBillboard()` method right above also be renamed then?

Reply:
sounds good to me